src/tls_wrap.cc

```diff
@@ -251,8 +244,6 @@ void TLSWrap::SSLInfoCallback(const SSL* ssl_, int where, int ret) {
   SSL* ssl = const_cast<SSL*>(ssl_);
   TLSWrap* c = static_cast<TLSWrap*>(SSL_get_app_data(ssl));
   Environment* env = c->env();
-  HandleScope handle_scope(env->isolate());
-  Context::Scope context_scope(env->context());
```

Comment:
These can be removed because SSLInfoCallback is called synchronously with a HandleScope and Context::Scope further down the stack?